src/main/resources/ath-container/Dockerfile

```diff
@@ -55,40 +56,46 @@ ARG GECKODRIVER_VERSION=0.34.0
 RUN arch=$(arch | sed s/aarch64/arm64/ | sed s/x86_64/amd64/) && \
   if [ "$arch" = "arm64" ] ; \
   then \
-    curl -fsSLO https://github.com/mozilla/geckodriver/releases/download/v${GECKODRIVER_VERSION}/geckodriver-v${GECKODRIVER_VERSION}-linux-aarch64.tar.gz && \
-    tar -xvzf geckodriver-v${GECKODRIVER_VERSION}-linux-aarch64.tar.gz -C /usr/local/bin; \
+    curl -fsSLO "https://github.com/mozilla/geckodriver/releases/download/v${GECKODRIVER_VERSION}/geckodriver-v${GECKODRIVER_VERSION}-linux-aarch64.tar.gz" && \
+    tar -xvzf "geckodriver-v${GECKODRIVER_VERSION}-linux-aarch64.tar.gz" -C /usr/local/bin; \
   else \
-    curl -fsSLO https://github.com/mozilla/geckodriver/releases/download/v${GECKODRIVER_VERSION}/geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz && \
-    tar -xvzf geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz -C /usr/local/bin ; \
+    curl -fsSLO "https://github.com/mozilla/geckodriver/releases/download/v${GECKODRIVER_VERSION}/geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz" && \
+    tar -xvzf "geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz" -C /usr/local/bin ; \
   fi
 
 # Maven in repo is not new enough for ATH
 ARG MAVEN_VERSION=3.9.8
-RUN curl -ffSLO https://archive.apache.org/dist/maven/maven-3/"${MAVEN_VERSION}"/binaries/apache-maven-"${MAVEN_VERSION}"-bin.tar.gz && \
-    tar -xvzf apache-maven-"${MAVEN_VERSION}"-bin.tar.gz -C /opt/ && \
-    mv /opt/apache-maven-* /opt/maven
-ENV PATH="$PATH:/opt/maven/bin"
+RUN curl -fsSLO "https://archive.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/binaries/apache-maven-${MAVEN_VERSION}-bin.tar.gz" && \
+    tar -xvzf "apache-maven-${MAVEN_VERSION}-bin.tar.gz" -C /opt/ && \
+    mv "/opt/apache-maven-${MAVEN_VERSION}" /opt/maven
+ENV PATH "$PATH:/opt/maven/bin"
 
 ENV SHARED_DOCKER_SERVICE true
 
+COPY run.sh /usr/bin
+COPY set-java.sh /usr/bin
+COPY vnc.sh /usr/bin
+
 # Allow injecting uid and git to match directory ownership
-ARG uid=1001
-ENV uid $uid
-ARG gid=1001
-ENV gid $gid
+ARG uid=1000
+ARG gid=1000
 
 EXPOSE 5942
 
-RUN deluser ubuntu && \
-    groupadd ath-user -g $gid && \
-    useradd ath-user -l -u $uid -g $gid -m -d /home/ath-user && \
-    # Give permission to modify the alternatives links to change the java version in use
-    chown ath-user:ath-user /etc/alternatives
+RUN deluser --remove-home ubuntu \
+  && groupadd ath-user -g $gid \
+  && useradd ath-user -l -c 'ATH User' -u $uid -g $gid -G docker -m -d /home/ath-user -s /bin/bash
```

Comment:
Adding a comment and setting the shell to `bash` to match the default Ubuntu user. Also adding `-G docker` so that we don't need to do any weird permissions changes later on for Docker-in-Docker.